stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.61.1
+version: 1.61.2
```

Comment:
I have a consider on the version, Should we bump the version to `2.0.0`?  Because of your PR  include the new feature and not in the backward-compatible right? That's mean if the user wants to make an upgrade to the new version, the upgradations would be failed.

Reply:
Its not a backward incompatable change, but it is a new feature? If so, 1.62.0.